packages/salesforcedx-vscode-apex/src/telemetry/telemetry.ts

```diff
@@ -79,6 +79,15 @@ export class TelemetryService {
     }
   }
 
+  public sendApexLSPLog(
+    properties?: { [key: string]: string },
+    measures?: { [key: string]: number }
+  ): void {
+    if (this.reporter !== undefined && this.isTelemetryEnabled) {
+      this.reporter.sendTelemetryEvent('apexLSPLog', properties, measures);
```

Comment:
Would it be useful to include the extension name here ?

Reply:
I thought it would be redundant since the eventName (apexLSPLog) is generated only by this extension.